DEVELOPMENT.md

```diff
@@ -12,7 +12,7 @@ After cloning suomifi-ui-components, run `yarn` to fetch its dependencies. Then,
 
 2. `yarn test` runs written tests.
 
-3. `yarn test:lint` checks TypeScript code for readability, maintainability, and functionality errors.
+3. `yarn lint` checks TypeScript code for readability, maintainability, and functionality errors.
```

Comment:
This change is unrelated to the actual subject matter of this PR, and we usually want to avoid that. But in this case it's fine because the change is so small and does not affect the library users (internal change)